src/xcode/ENA/ENA/Source/Services/HealthCertificateValidation/__tests__/BoosterNotificationsServiceTests.swift

```diff
@@ -57,190 +51,84 @@ class BoosterNotificationsServiceTests: XCTestCase {
 		]
 		
 		var resultRule: Rule?
-
 		boosterService.applyRulesForCertificates(certificates: certificates, completion: { result in
 			switch result {
 			case let .success(result):
 				resultRule = result.rule
-				expectation.fulfill()
 			case let .failure(error):
 				XCTFail("Test should not fail with error: \(error)")
 			}
+			expectation.fulfill()
```

Comment:
Just curios: Is there a benefit of fulfilling an expectation after calling XCTFail ?

Reply:
It's more about calling expectation.fulfill() after the switch over the result so it get's in any case executed (and it's easier for copying around the tests)